storage/local/interface.go

```diff
@@ -26,6 +26,8 @@ import (
 // are goroutine-safe. Storage implements storage.SampleAppender.
 type Storage interface {
 	prometheus.Collector
+	Querier
```

Comment:
Not sure I understand - how can we register the storage as a Collector anywhere when its interface doesn't have the required methods and all we return from `NewMemorySeriesStorage` is the interface?